packages/functions/src/dal/AppointmentDataAccessLayer.ts

```diff
@@ -131,6 +131,33 @@ export async function getAvailableAppointments() {
   }));
 }
 
+export async function getAppointmentsByStatus(
+  status: AppointmentStatus,
+  fromTime?: Date,
+  toTime?: Date
```

Comment:
Please follow the pattern in line 58
`options: { fromTime?: Date; toTime?: Date }`

Reply:
getAppointmentsByHospital also not consistent?

Reply:
Right. 

What ever you think is best